kedro/config/omegaconf_config.py

```diff
@@ -266,6 +281,22 @@ def _check_duplicates(seen_files_to_keys: Dict[Path, Set[Any]]):
             dup_str = "\n".join(duplicates)
             raise ValueError(f"{dup_str}")
 
+    @staticmethod
+    def _resolve_environment_variables(config: Dict[str, Any]) -> None:
+        """Use the ``oc.env`` resolver to read environment variables and replace
+        them in-place, clearing the resolver after the operation is complete if
+        it was not registered beforehand.
+
+        Arguments:
+            config {Dict[str, Any]} -- The configuration dictionary to resolve.
+        """
+        if not OmegaConf.has_resolver("oc.env"):
+            OmegaConf.register_new_resolver("oc.env", oc.env)
```

Comment:
Maybe we can drop the `oc.` or simply support both `env` and `oc.env`?

Reply:
Do you mean drop the "oc." in:
1.  name of the resolver (first argument to `register_new_resolver`)
or in:
2. the name assigned to `oc.env` (second argument to `register_new_resolver`)?

I personally wouldn't drop the `oc.` from 1., since this is the name assigned to the resolver by `OmegaConf` when it is instantiated. For 2. I would be ok with losing the namespace from the import: `from omegaconf.resolvers.oc import env`.